packages/@ember/-internals/glimmer/tests/integration/event-dispatcher-test.js

```diff
@@ -420,12 +420,15 @@ moduleFor(
     constructor() {
       super(...arguments);
 
-      let dispatcher = this.owner.lookup('event_dispatcher:main');
-      run(dispatcher, 'destroy');
-      this.owner.__container__.reset('event_dispatcher:main');
```

Comment:
I was not able to keep this test unchanged, as the reference to the `EventDispatcher` of the Glimmer Environment (already introduced in the initial PR) would always refer to the first instance, not the second once created here. So the curly component manager would still see the old dispatcher, that has no custom events added.

But I think this affects only our internal testing here, and not imply any publicly visible changes.